### PR TITLE
Enable wake to build against wasm target

### PR DESCRIPTION
**Pull request by V-FEXrt**

### CODE REVIEW

**Comment by V-FEXrt** [comment]
I wasn't able to test this since I still can't build without gmp.h, but this removed the curses build error

**Review by JakeSiFive — APPROVED** [review]

**Comment by JakeSiFive** [comment]
> I wasn't able to test this since I still can't build without gmp.h, but this removed the curses build error

Yep, I'll fix this tommarow

**Review by JakeSiFive — APPROVED** [review]
Not sure if this is the working version but I'm happy with all the code here

File: .circleci/config.yml [hunk 1]
```diff
@@ -212,6 +223,9 @@ workflows:
       - vscode:
           requires:
             - tarball
+      - wasm:
+          requires:
+            - tarball
```
Comment (JakeSiFive) [inline .circleci/config.yml:228]: Oh I got this wrong. I assumed vscode didn't depend on tarball. Still given that it does this is the right outcome.

**Review by JakeSiFive — APPROVED** [review]

**Pull request merged**